modules/openapi-generator/src/main/resources/Java/JSON.mustache

```diff
@@ -2,11 +2,11 @@
 
 package {{invokerPackage}};
 
+import com.fasterxml.jackson.databind.util.StdDateFormat;
```

Comment:
This generates a client that does not compile for me. I don't have any need for supporting java.sql.Date but now I have to import jackson for stuff to compile. For now we are going to use a custom mustache template which is the old version I guess.